server/models/Collective.js

```diff
@@ -308,6 +308,11 @@ export default function(Sequelize, DataTypes) {
         defaultValue: false,
       },
 
+      approvedAt: {
+        type: DataTypes.DATE,
+        defaultValue: Sequelize.NOW,
+      },
+
```

Comment:
In the spec, we said we wanted a behavior similar `isArchived`.

We wanted to expose `isApproved` (a boolean), not `approvedAt` (a date).

Reply:
@znarf But this is the model, not the graphql endpoint.

Reply:
@znarf But this is the model, not the graphql endpoint.